README.md

```diff
@@ -242,7 +242,7 @@ curl -H "x-retry-test-id: 1d05c20627844214a9ff7cbcf696317d" "http://localhost:91
 | Failure Id              | Description
 | ----------------------- | ---
 | return-X                                  | Testbench will fail with HTTP code provided for `X`, e.g. return-503 returns a 503
-| return-X-after-YK                         | Testbench will return X after YKiB of uploaded data
-| return-broken-stream-final-chunk-after-YB | Testbench will break connection on final chunk of a resumable upload after Y bytes.
+| return-X-after-YK                         | Testbench will return X after YKiB of uploaded or downloaded data
```

Comment:
nit: `return-X-after-YK` cannot really work for a download... the error code is returned at the beginning, after the headers but before the payload.  Any client will receive a 200 if the download starts successfully.

Reply:
I see. I did not think of the fact that the error code is returned before the payload. So `return-X-after-YK` is unrealistic to happen for downloads in the live server...

I think it "works" here merely because I forcefully trigger the streamer first before returning the response (L694)
```
# Inject fault outside of the streamer considering execution flow and werkzeug wrappers
try:
    response = flask.Response(streamer())
    _ = response.data
    return response
except testbench.error.RestException as e:
    raise e
 ```
Thus it "works" when I test it against the python retry conformance test, like so
```
127.0.0.1 - - [02/Feb/2022:16:50:50 -0800] "GET /download/storage/v1/b/aef6bb4577734e59b6f41c5cdfe501fa/o/4678f04722e54979845541c381a5025e?alt=media HTTP/1.1" 504 97 "-" "python-requests/2.27.1"
127.0.0.1 - - [02/Feb/2022:16:50:51 -0800] "GET /download/storage/v1/b/aef6bb4577734e59b6f41c5cdfe501fa/o/4678f04722e54979845541c381a5025e?alt=media HTTP/1.1" 200 1100000 "-" "python-requests/2.27.1"
```

I will probably close this FR since it does not match live server behavior, unless we decide it is helpful in an way for testing. Thanks for the feedback!